src/builtin/cd_command.ts

```diff
@@ -5,9 +5,29 @@ import { IRunContext, ITabCompleteContext } from '../context';
 import { GeneralError } from '../error_exit_code';
 import { ExitCode } from '../exit_code';
 import { ITabCompleteResult, PathType } from '../tab_complete';
+import { BooleanArgument } from '../argument';
 
 class CdArguments extends CommandArguments {
   positional = new PositionalPathArguments({ pathType: PathType.Directory });
+  help = new BooleanArgument('h', 'help', 'display this help and exit');
+
+  constructor() {
+    super({
+      description: `Change the shell working directory.
+    
+    Change the current directory to DIR.  The default DIR is the value of the
```

Comment:
We need to careful to only add help that is correct. `cd` without any arguments does not `cd` to the `HOME` directory, and there is no use of `CDPATH`, and no shell option `cdable_vars`. So they should not be mentioned.